README.md

```diff
@@ -1 +1,70 @@
-REACT TO DO APP
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.\
```

Comment:
Nice work you've done so far in this project, but you need to add a more descriptive readme file

Reply:
I can see that you have also not implemented the Navbar component as required. you need to work on this

Reply:
@Divinity-dev  Hello and thank you for the comment, I will definitely work on the mentioned parts, though I deliberately choose not to do the hamburger nav bar area since it was optional
- Take a look at the image below, but if I find time I will surely work on it, thank you.
-
![Screenshot from 2023-03-02 16-03-41](https://user-images.githubusercontent.com/55842414/222436842-6ecdca84-24a8-41ff-9297-318692ca4ec5.png)